app/models/playbook/config_type.rb

```diff
@@ -0,0 +1,38 @@
+class Playbook::ConfigType < ActiveRecord::Type::Value
+  attr_accessor :backstory, :headings
+  # questions - questions we ask player
+  # options - question options
+
+  def cast(value)
+    return Playbook::ConfigType.new if value.nil?
+    return value if value.is_a?(Playbook::ConfigType)
+    value = JSON.parse(value, symbolize_names: true) if value.is_a?(String)
+    self.backstory = value[:backstory]
+    self.headings = backstory[:headings]
+    self
+  end
+  
+  def serialize(value)
+    return nil if value.nil? || value.empty?
+    value.to_hash

```

Comment:
We do want this to return a string
```suggestion
    value.to_hash.to_json

```